tom/dependencies.py

```diff
@@ -129,7 +135,10 @@ def maybe_replace(self, string, match, old, new):
 
     def extract_version_from_filename(self, dep, filename):
         if dep == 'openssl':
-            version = re.search('-([0-9a-z.]*).tar', filename).group(1)
+            try:
+                version = re.search('-([0-9a-z.]*).tar', filename).group(1)
+            except:
+                version = re.search('_([0-9a-z_]*).tar', filename).group(1)
```

Comment:
Please do just `re.search`, check the result, then try the other one if it is `None` and then do `group(1)` on the match.